### PR TITLE
add comment with Startup project dropdown help

### PR DESCRIPTION
**Pull request by TheBottleCyber**
Created to solve #114

### CODE REVIEW

**Comment by TheBottleCyber** [comment]
@seclerp I guess I did something different than what you wanted, but I meant that I would write an md for it, and the label I thought you would do 😨.

**Review by seclerp — CHANGES_REQUESTED** [review]
The majority of the requirements listed here are not about "Startup project". Let's not duplicate README, we are only interested in troubleshooting Startup project selection.

The project should satisfy 2 conditions:
1. `Microsoft.EntityFrameworkCore.Design` or `Microsoft.EntityFrameworkCore.Tools` NuGet package is installed.
2. Project's `TargetFramework` is at least `netcoreapp3.1`.

They both are presented in the F.A.Q. page. Do you have something to add regarding Startup project especially?

**Comment by seclerp** [comment]
Do you like to try implementing #114 on the UI side? It should be quite easy if you're interested.

**Comment by TheBottleCyber** [comment]
> Do you like to try implementing #114 on the UI side? It should be quite easy if you're interested.

I will. As soon as I'm done with the .md

**Comment by TheBottleCyber** [comment]
Do you want the md file only on https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field or I can use the way I did

**Comment by seclerp** [comment]
I don't see anything new here regarding the Startup project requirements for dropdown. So let's then proceed to the UI implementation with the link to the F.A.Q. page.

If you are looking for a usage tutorial, we already have a full-featured [tutorial in the JetBrains blog](https://blog.jetbrains.com/dotnet/2022/01/31/entity-framework-core-inside-rider-ui-way) and a 222.1 "What's new" blog post [here](https://blog.seclerp.me/general/ef-core-inside-rider-large-update/).

**Comment by TheBottleCyber** [comment]
Ok, can you update the F.A.Q. around this content?

## Why I can't see my project in a "Startup projects" field?

### Requirements to become a Startup project
- `Microsoft.EntityFrameworkCore.Design` or `Microsoft.EntityFrameworkCore.Tools` NuGet package is installed.

### Check up before using plugin
- You have global tools (`dotnet ef`) with right version
- You have project contains `DatabaseContext` class that **inherits** `DbContext`
- You have project which will be `Startup project` by have reference to **migration tool package**

> **Note**: One project can be **Migration project**, **Startup project** and contains **DatabaseContext** class

**Comment by seclerp** [comment]
Let's proceed with each item one by one:
> - You have global tools (`dotnet ef`) with right version

We have a notification on the right bottom corner when it's not installed. Moreover, it's the first thing that you should do in any EF Core tutorial. Again, the basics of EF Core is NOT in a scope of this project. See [EF Core Getting Started](https://docs.microsoft.com/en-us/ef/core/get-started/overview/first-app?tabs=netcore-cli) guide from Microsoft team.

> - You have project contains DatabaseContext class that inherits DbContext

Again, see [the guide](https://docs.microsoft.com/en-us/ef/core/get-started/overview/first-app?tabs=netcore-cli)

> - You have project which will be Startup project by have reference to migration tool package

I can't understand what you mean by this.

**Comment by TheBottleCyber** [comment]
@seclerp seems to be done with https://github.com/seclerp/rider-efcore/pull/115/commits/8f57cae63ad62c0174fdb974486519d31bb66449

File: src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/shared/BaseDialogWrapper.kt [hunk 1]
```diff
@@ -241,6 +241,7 @@ abstract class BaseDialogWrapper(
             iconComboBox(availableStartupProjects, { commonOptions.startupProject }, ::startupProjectSetter)
                 .validationOnInput(validator.startupProjectValidation())
                 .validationOnApply(validator.startupProjectValidation())
+                .comment("Your project is not listed? <a href='https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field'>Help</a>")
```
Comment (seclerp) [inline src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/shared/BaseDialogWrapper.kt:244]: A little suggestion:

```suggestion
                .comment(
                    "Your project is not listed? " +
                    "<a href='https://plugins.jetbrains.com/plugin/18147-entity-framework-core-ui/f-a-q#why-i-cant-see-my-project-in-a-startup-projects-field'>" +
                        "Help" +
                    "</a>")
```

**Review by seclerp — APPROVED** [review]

**Review by seclerp — APPROVED** [review]

**Comment by seclerp** [comment]
LGFM, thanks!

**Pull request merged**